### ISSUE TITLE
Investigate why iframe-helper test fails when test iframes are failing

### ISSUE THREAD

**Issue by mkhatib** [ampproject/amphtml#3314]
``` sh
Chrome 50.0.2661 (Mac OS X 10.11.5) iframe-helper should un-listen on next message when iframe is unattached FAILED
    AssertionError: expected 1 to be above 4
        at /var/folders/7w/bvjck7ls7f58m040x_qvph4w002y5_/T/524b011d895891229e9af880da84620e.browserify:74212:34 <- /Users/mkhatib/amphtml/test/functional/test-iframe-helper.js:130:33
```

**Comment by adelinamart** [comment]
Hey,

The AMP community has been working nonstop to make AMP better, but somehow we've still managed to grow an enormous backlog of open issues.  This has made it difficult for the community to prioritize what we should work on next.

A new process is on the way and to give it a chance for success we will be closing issues that have not been updated in awhile.

If this issue still requires further attention, simply reopen it.  Please try to reproduce it with the latest version to ensure it gets proper attention!

We really appreciate the contribution! Thank you for bearing with us as we drag ourselves out of the issue abyss. :)

**Comment by aghassemi** [comment]
closing as stale.
